src/buffer.c

```diff
@@ -189,7 +189,7 @@ int BUFFER_build(BUFFER_HANDLE handle, const unsigned char* source, size_t size)
             {
                 b->buffer = newBuffer;
                 b->size = size;
-                /* Codes_SRS_BUFFER_01_002: [The size argument can be zero, in which case nothing shall be copied from source.] */
+                /* Codes_SRS_BUFFER_01_002: [The size argument can be zero, in which case the underlying buffer held by the buffer instance shall be freed.] */
                 (void)memcpy(b->buffer, source, size);
```

Comment:
The spec SRS_BUFFER_01_002 is already tagged in line 159, so it should not be tagged twice mistakenly here